src/timepicker/timepicker.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+
+@Component({
+	selector: "ibm-timepicker",
+	template: `
+		<div class="bx--form-item">
```

Comment:
Is this needed? Can this be applied to host instead?

Would it make sense to take this out and put `bx--time-picker` on host so when they use it, they would put the component in a form item if they want to, otherwise host would be what current div.bx--time-picker is?

Reply:
fixed :) thank you for pointing it out